draft-ietf-rats-architecture.md

```diff
@@ -813,6 +813,9 @@ component or user on the device.
 
 ## Attestation Results
 
+Attestation Results are the input used by the Relying Party to decide whether it will trust a particular device, allow access to some content or data, allow a financial transaction or other.
```

Comment:
```suggestion
Attestation Results are the input used by the Relying Party to decide the extent to which it will trust a particular Attester, and allow it to access some data or perform some operation.
```